modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Classes\Render;
+
+use Elementor\Utils;
+
+/**
+ * Class Core_Render.
+ *
+ * This class handles the rendering of the Link In Bio widget for the core version.
+ *
+ * @since 3.23.0
+ */
+class Core_Render extends Render_Base {
+
+	public function render( \Elementor\Modules\ConversionCenter\Widgets\Link_In_Bio $widget ): void {
+		$settings = $widget->get_settings_for_display();
+
+		/**
+		 * Map args to internals
+		 */
+
+		//  Identity Image
+		$identity_image_props_style = $settings['identity_image_style'] ?? 'profile';
+		$identity_image_value       = $settings['identity_image'] ?? [];
```

Comment:
```suggestion
		$identity_image_value = $settings['identity_image'] ?? [];
```

Reply:
Yeah we don't normally do this, but I guess it's okay if it's easier to read for you.